src/Stack/Options.hs

```diff
@@ -294,6 +303,15 @@ configOptsParser hide0 =
             hide
   where hide = hideMods (hide0 /= OuterGlobalOpts)
 
+readAbsDir :: ReadM (Path Abs Dir)
```

Comment:
`ReadM` sadly isn't an instance of `MonadThrow`, which `parseAbsDir` requires.

That might change with https://github.com/pcapriotti/optparse-applicative/pull/196 though.